ethereum/contracts/ERC20App.sol

```diff
@@ -69,7 +70,8 @@ contract ERC20App {
 
         bytes memory call = encodeCall(_token, msg.sender, _recipient, _amount);
 
-        OutboundChannel channel = OutboundChannel(channels[_channelId].outbound);
+        OutboundChannel channel =
```

Comment:
Small q - Is it recommended for Solidity code to not go over 80 columns? As I see both you and @musnit seem to be following that.

Reply:
i think @denalimarsh just copied my code and my setup is doing this